docs/topics/feed-exports.rst

```diff
@@ -267,10 +267,129 @@ once all the file contents have been written (i.e. at the end of the crawl) is
 that file uploaded to the feed URI.
 
 If you want item delivery to start earlier when using one of these storage
-backends, use :setting:`FEED_EXPORT_BATCH_ITEM_COUNT` to split the output items
-in multiple files, with the specified maximum item count per file. That way, as
-soon as a file reaches the maximum item count, that file is delivered to the
-feed URI, allowing item delivery to start way before the end of the crawl.
+backends, use :ref:`batches` to split the output items into multiple
+files, delivered sequentially after a given criteria is met, e.g. after each
+file contains a given number of items or after a given time has passed since
+the last file was delivered.
+
+.. _batches:
+
+Batches
+=======
+
+.. versionadded:: VERSION
+
+Batches enable multiple sequential files to be generated for a feed instead of getting
+a single file. This is useful in getting item deliveries earlier before the crawl ends.
+
+Batching is controlled by a batch handler, which is defined through the 
+:ref:`batch feed option <feed-options>`.
+For information about enabling and configuring the default batch handler, see
+:class:`~scrapy.extensions.batches.BatchHandler`. You can alternatively 
+:ref:`create your own batch handler <custom-batch-handler>`.
+
+If you enable batching, you must also :ref:`set feed URIs appropriately with placeholders<batch-uri>`.
+
+.. _default-batch-handler:
+
+Default Batch Handler
+----------------------
+
+.. autoclass:: scrapy.extensions.batches.BatchHandler
+
+.. _custom-batch-handler:
+
+Custom Batch Handler
+--------------------
+
+To use a custom batch handler, write a custom batch handler class and assign 
+that class or its import string to the :ref:`batch feed option <feed-options>`.
+
+Each batch handler is a class that must implement the following methods:
+
+.. method:: __init__(self, feed_options)
```

Comment:
I think you need to add `:noindex:` to every `method`, `class`, and other API entries that do not describe something that is part of Scrapy, but an interface for users to implement themselves. For example:

```suggestion
.. method:: __init__(self, feed_options)
   :noindex:
```

Alternatively, you could wrap them as part of a class (e.g. `.. class:: MyBatchHandler`). I think that’s how we have done it in similar cases in the past. Although if `:noindex:` works, maybe that would be best. Up to you.